### PR TITLE
Demos for Scala Days 2018

### PR DESCRIPTION
**Pull request by vjovanov**

### CODE REVIEW

**Comment by vjovanov** [comment]
Can we merge this PR?

**Comment by vjovanov** [comment]
Guys, can we merge this?

File: scala-days-2018/streams-pgo/TestStream.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    getValue(persons);
+   }
+   long currentTime = (System.currentTimeMillis() - time);
+   sum += currentTime;
+   System.out.println("Iteration " + j + " finished in " + currentTime + " milliseconds.");
+  }
+  System.out.println("TOTAL time: " + sum);
+ }
+
+ public static double getValue2() {
```
Comment (christianwimmer) [inline scala-days-2018/streams-pgo/TestStream.java]: What is this code for? Seems unused and also not mentioned in the readme (same for the non-PGO file).

Do we really want the same file committed twice? Why not just have two .md files in the same directories to explain the non-PGO and the PGO part?

**Comment by christianwimmer** [comment]
I don't think I have write access to this repository.

File: scala-days-2018/streams-pgo/TestStream.java (hunk 1 above)
Comment (vjovanov) [reply scala-days-2018/streams-pgo/TestStream.java]: This must be testing code that I left by accident. I'll fix this in the morning.

**Pull request merged**